modules/policy-identity-access-management.adoc

```diff
@@ -49,8 +49,19 @@ In AWS, SREs generate a short-lived AWS access token for the `BYOCAdminAccess` u
 
 In Google Cloud, SREs access resources after being authenticated against a Red Hat SAML identity provider (IDP). The IDP authorizes tokens that have time-to-live expirations. The issuance of the token is auditable by corporate Red Hat IT and linked back to an individual user.
 
+[id="osd-sre-sts-view-aws-account_{context}"]
+== SRE STS view of AWS accounts
+
+When SRE is on VPN through two-factor authentication, Red Hat Support and SRE can assume the `ManagedOpenShift-Support-Role` in your AWS Account. `ManagedOpenShift-Support-Role` has all the permissions necessary for SRE to troubleshoot AWS resources. Upon assumption of the `ManagedOpenShift-Support-Role`, SRE uses a AWS Security Token Service (STS) to perform troubleshooting actions in your account. SRE can perform multiple actions which include:
+
+* Viewing CloudTrail logs
+* Shutting down a faulty EC2 Instance
+* Creating EC2 snapshots
+
+For a full list of permissions, see sts_support_permission_policy.json in the link:https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html[About IAM resources for ROSA clusters that use STS] user guide.
```

Comment:
Just one note - do we have a corresponding link for OSD?

Reply:
I am removing the OSD content. OSD doesn't support STS.

Reply:
Also, please see [slack thread](https://redhat-internal.slack.com/archives/C05Q2MLFB0X/p1694093293402269?thread_ts=1694003063.601879&cid=C05Q2MLFB0X)

Cc @karanthshashank @laujohns89